data/ThorOnTheRocks.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Gianluca Galota",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/ThorOnTheRocks.png",
+  "tags": ["javascript", "typescript", "react", "node", "open source", "design system"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/GianlucaGalota" },
+    { "platform": "github", "url": "https://github.com/ThorOnTheRocks" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ThorOnTheRocks",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/GianlucaGalota",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/gianluca-galota/",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Joined EddieHub",
+      "date": "November 2022",
+      "icon": "github",
+      "color": "blue",
+      "description": "Made my first PR to EddieHub and my first contribution to open source",
+      "url": "https://github.com/EddieHubCommunity/LinkFree"
+    },
+    {
+      "title": "Started my journey as Frontend Developer",
+      "date": "November 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "First job as Frontend Developer",
+      "url": "https://thorontherocks.github.io/personal-website/"
```

Comment:
There is a broken URL. Please fix the URL with the correct one! 🛠

![image](https://user-images.githubusercontent.com/88786642/204327855-7a6a3d89-123e-4b5e-be55-b1191bc8edd6.png)